framework/src/util/Properties.cpp

```diff
@@ -158,6 +158,58 @@ std::pair<Any, bool> Properties::Value_unlocked(const std::string& key,
   }
 }
 
+const Any& Properties::Value(const std::string& key, bool matchCase) const
```

Comment:
This function should be named something more descriptive, e.g. `RefToValue`, `ValueByRef`, etc...